module/main.py

```diff
@@ -2,57 +2,52 @@
 
 
 class Figures:
-    def __init__(self, x, t):
-        self.__x = x
-        if x == 1:
-            self.Line = line.Line(int(input('Enter line size:\n')),t)
+    def __init__(self, figure, i, t):
+        w = []
```

Comment:
who stole your letters? What is this kind of variable namings? It's okay to have i,j iterators but what is W? I don't know this kind of naming conventions